src/actor.js

```diff
@@ -69,6 +70,24 @@ const waitForRunToFinish = async ({ actId, runId, token, waitSecs, taskId }) =>
     return updatedRun;
 };
 
+/**
+ * Parses input and contentType and appends it to a given options object.
+ * Throws if input is not valid.
+ *
+ * @ignore
+ */
+const addInputOptionsOrThrow = (input, contentType, options) => {
+    options.contentType = contentType;
+
+    // NOTE: this function modifies contentType property on options object if needed.
+    options.body = maybeStringify(input, options);
+
+    checkParamOrThrow(options.body, 'input', 'Buffer|String');
+    checkParamOrThrow(options.contentType, 'contentType', 'String');
+
+    if (options.contentType) options.contentType = addCharsetToContentType(options.contentType);
```

Comment:
Is this condition needed if it's already checked two lines above (or is it ok if content type is empty string?)

Reply:
good point